src/ctap2/commands/get_assertion.rs

```diff
@@ -67,17 +68,100 @@ impl UserVerification for GetAssertionOptions {
 }
 
 #[derive(Debug, Clone)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
Again, `PartialEq` could probably be derived always, unless there is a reason not to. Same below with `HmacGetSecretOrPrf`.

Reply:
Done! https://github.com/mozilla/authenticator-rs/pull/337/commits/093957b831de339a0b971f91f6aa560353c0ab62